README.md

```diff
@@ -16,6 +16,7 @@ Here is a  ~5 minutes demo  of the operator.
 
 ### Building the image
 
+*Note: This step is optional and needs to be run only if you have custom changes in the code.*
```

Comment:
Hmm, but providing helm chart or bunch of yaml for users to do a `helm install` is not that different from `make deploy` which calls `kubectl apply` on the same yaml files in both cases.
I think it comes down to the default value - whether or not to default it to quay.io/redhat-pipeline-service.

But I do see another risk here. Users (say our team members) who do have full rights on the quay repo can accidentally push their changes. I think its best to avoid such a situation. I will put the default to settings-operator to maintain consistency with the name.

Reply:
The difference is whether the user needs to do a `git clone` or not. Non dev users should not need to do that but it is no big thing and as you highlight the push risk is of bigger concern.

Reply:
Reverted to 'settings-operator' image instead of the quay.io one